fury/actor.py

```diff
@@ -3795,3 +3796,71 @@ def callback(
     shader_to_actor(sq_actor, 'fragment', impl_code=fs_impl_code, block='light')
 
     return sq_actor
+
+
+def tensor(
```

Comment:
we will need to talk with @guaje and @Garyfallidis concerning the naming. 

or, we need to find a way to explain what is the difference between `tensor` and `tensor_slicer`

Reply:
In the end, we decided to go with `ellipsoid` to keep the actor more generic. Additionally, I will work on a tutorial to show the use that can be made of this new actor in the visualization of diffusion tensor ellipsoids, in comparison with the `tensor_slicer` actor.